cmd/cli/kubectl-kyverno/processor/policy_processor.go

```diff
@@ -377,13 +380,34 @@ func (p *PolicyProcessor) printOutput(resource interface{}, response engineapi.E
 		return fmt.Errorf("failed to marshal (%w)", err)
 	}
 
+	var yamlEncodedTargetResources [][]byte
+	for _, ruleResponese := range response.PolicyResponse.Rules {
+		patchedTarget, _, _ := ruleResponese.PatchedTarget()
+
+		if patchedTarget != nil {
+			yamlEncodedResource, err := yamlv2.Marshal(patchedTarget.Object)
+			if err != nil {
+				return fmt.Errorf("failed to marshal (%w)", err)
+			}
+
+			yamlEncodedResource = append(yamlEncodedResource, []byte("\n---\n")...)
+			yamlEncodedTargetResources = append(yamlEncodedTargetResources, yamlEncodedResource)
+		}
+	}
+
 	if p.MutateLogPath == "" {
 		resource := string(yamlEncodedResource) + string("\n---")
 		if len(strings.TrimSpace(resource)) > 0 {
 			if !p.Stdin {
 				fmt.Fprintf(p.Out, "\npolicy %s applied to %s:", response.Policy().GetName(), resourcePath)
 			}
 			fmt.Fprintf(p.Out, "\n"+resource+"\n") //nolint:govet
+			if len(yamlEncodedTargetResources) > 0 {
+				fmt.Fprintf(p.Out, "patched targets: \n")
```

Comment:
Did we add the option to generate a clean output?

Reply:
If by this you mean creating a file output that just prints the resources then yes
In `cmd/cli/kubectl-kyverno/processor/policy_processor.go:439`:
```golang
	for _, patchedTarget := range yamlEncodedTargetResources {
		if _, err := file.Write(patchedTarget); err != nil {
			return err
		}
	}
	if err := file.Close(); err != nil {
		return err
	}
```

If you would like to add a flag that makes you print resources to standard out without anything else thats doable

Reply:
Output to a file is good enough, what is the flag?

Reply:
`-o`
The same flag that outputs trigger resource mutations to a file. its appropriate to have it output all mutations to triggers and targets alike